Automattic/jetpack
# Make Variations for Payment Block Visible on Click

Fixes #28523

## Proposed changes:

There may be times when the inner blocks rendered by the Payment block are hidden. Update the visibility setting of the inner block on click if the block is hidden

https://user-images.githubusercontent.com/7076981/231888432-3414b821-8756-4ca1-9d8c-1306398153ff.mov



### Other information:


## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Sync changes to your sandbox
* Disable the Donation block in Preferences > Blocks ( see video above )
* Insert the Payments block
* Select the Donations block
* Notice the Donations block is displayed
